TAScheduler/views.py

```diff
@@ -277,6 +279,65 @@ def post(self, request):
                               {"message": str(e), "courses": courses})
 
 
+class InstructorAssignTaToSection(View):
+
+    def get(self, request):
+        if request.session.get("user") is None:
+            return redirect("/")
+        if determineUser(request.session["user"]).getRole() != "Instructor":
+            return redirect("/home/")
+        tas = list(map(str, TA.objects.all()))
+        sections = list(map(str, Lab.objects.all()))
+        sections.extend(list(map(str, Lecture.objects.all())))
+        if len(tas) == 0:
+            return render(request,
+                          "error.html",
+                          {"message": "No TAs to display", "previous_url": "/home/managesection/"})
+        if len(sections) == 0:
+            return render(request,
+                          "error.html",
+                          {"message": "No Sections to display", "previous_url": "/home/managesection/"})
+
+        return render(request, "sectionManagement/instructor_assign_ta_to_section.html",
+                      {"tas": tas, "sections": sections, "message": "Please select a ta to assign to a section"})
+
+    def post(self, request):
+        selecteduser = determineUser(request.POST.get("ta"))
+        selectedsection = determineSec(request.POST.get("section"))
+        if selecteduser is None or selecteduser == '':
+            tas = list(map(str, TA.objects.all()))
```

Comment:
since this logic is repeated 3 times (getting all sections + tas), maybe make a helper method to make it look better